apps/console/src/features/groups/pages/groups.tsx

```diff
@@ -421,16 +428,18 @@ const GroupsPage: FunctionComponent<any> = (): ReactElement => {
                 onPageChange={ handlePaginationChange }
                 onSortStrategyChange={ handleListSortingStrategyOnChange }
                 sortStrategy={ listSortingStrategy }
-                rightActionPanel={
-                    (<Dropdown
-                        data-testid="group-mgt-groups-list-stores-dropdown"
-                        selection
-                        options={ userStoreOptions && userStoreOptions }
-                        placeholder={ t("console:manage.features.groups.list.storeOptions") }
-                        value={ userStore && userStore }
-                        onChange={ handleDomainChange }
-                    />)
-                }
+                rightActionPanel={ (
+                    <RootOnlyComponent>
+                        <Dropdown
+                            data-testid="group-mgt-groups-list-stores-dropdown"
+                            selection
+                            options={ userStoreOptions && userStoreOptions }
```

Comment:
What would happen if userStoreOptions are not present at the time of this dropdown rendering? IMO, the better way would be to conditionally render it. 

Eg: 

```js
{ userStoreOptions.length > 0 && (
    <Dropdown
        data-testid="group-mgt-groups-list-stores-dropdown"
        selection
        options={ userStoreOptions }           
         placeholder={ t("console:manage.features.groups.list.storeOptions") }
         onChange={ handleDomainChange }
         defaultValue={ GroupConstants.ALL_USER_STORES_OPTION_VALUE }
    />
)}
```

Reply:
`userStoreOptions` array has 2 elements by default[1].

[1] https://github.com/wso2/identity-apps/blob/f6965d2d33b7d6f46f938122e3368f932b54b36c/apps/console/src/features/groups/pages/groups.tsx#L188-L199